mission/functions/systems/sites/utils/fn_sites_utils_normalise_object_placement.sqf

```diff
@@ -0,0 +1,24 @@
+/*
+	File: fn_sites_utils_normalise_object_placement.sqf
+	Author: 'DJ' Dijksterhuis
+	Public: No
+	
+	Description:
+		
```

Comment:
Needs a description. Something like

```
1. Sets up an object with the appropriate 'up' vector for the current terrain (i.e. sets the object rotation so it will be perpendicular to the instantaneous terrain gradient)/
2. Sets the AGLS z-coordinate of the object to 0.1 -- objects might be placed on terrain which has curvature, necessitating the object be placed slightly above the terrain itself.

```